ArduCopter/system.pde

```diff
@@ -399,6 +399,24 @@ static void frsky_telemetry_send(void)
 #endif
 
 /*
+  check a digitial pin for high,low (1/0)
+ */
+static uint8_t check_digital_pin(uint8_t pin)
```

Comment:
Hi @jmachuca77 ,
This function already was in Rover, indeed is a copy-paste from it. Originally was used for AUTO_TRIGGER_PIN.